### PR TITLE
335 secondary links + refactoring source link code

### PR DESCRIPTION
**Pull request by ayya-vimala**
After uploading of new Sanskrit files some of the formatted segment links still need to be looked at.

### CODE REVIEW

File: src/views/utility/source-link.js [hunk 1]
```diff
@@ -44,87 +48,112 @@ export class FormattedFileName extends LitElement {
           padding-bottom: 6px;
           padding-left: 4px;
         }
+
+        .cbclink {
+          background-color: #333333;
```
Comment (sebastian-nehrdich) [inline src/views/utility/source-link.js:53]: this is all very nice code and works great as far as I tested it, the only thing I am not sure about is the background color for the CBC link. Maybe we should just go for the same color as the button? the dark background right now is a bit disturbing (at least I think so).

Comment (ayya-vimala) [reply src/views/utility/source-link.js:53]: I made it the same color as they are using. But I can take all that css code out. We do not need to do the same thing they are doing as they don't have an official logo.

**Comment by ayya-vimala** [comment]
How is this?

**Comment by sebastian-nehrdich** [comment]
good!

**Review by sebastian-nehrdich — APPROVED** [review]

**Pull request merged**